core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -168,8 +170,17 @@ oppia.factory('trainingModalService', [
                   editorContextService.getActiveStateName();
                 var state = explorationStatesService.getState(currentStateName);
 
+                // Retrieve the interaction ID.
+                var interactionId = stateInteractionIdService.savedMemento;
+
+                var rulesServiceName = interactionId.charAt(0).toLowerCase() +
+                interactionId.slice(1) + 'RulesService';
```

Comment:
@seanlip done! I haven't moved anything old into the new service. Also, I have added the specs. Please let me know if it looks alright or not. Thanks!